frontend/src/views/WheresMyGene/common/store/actions.ts

```diff
@@ -200,8 +200,13 @@ export function setXAxisHeight(
   };
 }
 
+interface SetFilteredCellTypesPayload {
```

Comment:
not sure if this is the best place to declare this custom payload and importing it in the reducer